src/icalendar/fuzzing/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -eu
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Since this file is under the apache license, we need to check that we do not ship it or adjust the license information.

Reply:
I think, yes that would be a fix.

Reply:
I'm not familiar with how to prevent that. Any suggestions?

Reply:
I can not test it at the moment but ciis is what I found: https://stackoverflow.com/questions/43365915/python-packaging-exclude-directory-from-bdist-wheel